docs/external-registry-ca.md

```diff
@@ -0,0 +1,39 @@
+# External Registry CA Injection
+
+If you are using a secure external container registry to hold mirrored OpenShift
+release images, Cincinnati will need access to this registry in order to build
+an upgrade graph.  Here's how you can inject your CA Cert into the Cincinnati
+pod.
+
+OpenShift has an external registry API, located at `image.config.openshift.io`,
+that we'll use to store the external registry CA Cert.  You can read more about
+this API in the [OpenShift documentation](https://docs.openshift.com/container-platform/4.3/registry/configuring-registry-operator.html#images-configuration-cas_configuring-registry-operator).
+
+Create a ConfigMap in the `openshift-config` namespace.  Fill in your CA Cert
+and ConfigMap name, but keep the key, `cincinnati-registry`, because it's how
```

Comment:
based on the implementation, it appears extremely important for the `ConfigMap` name to be `trusted-ca`.

Reply:
You have to create a configmap with the same name that was added to `ImageConfig.Spec.AdditionalTrustedCA.Name`.  The operator watches for a configmap named whatever is in `ImageConfig.Spec.AdditionalTrustedCA.Name`, so it should accept other names.

Reply:
I think this might read a little more clearly like this: "Fill in your CA Cert under the key `cincinnati-registry`." without talking about the name. I think it's implied below that the name of the confimap is up to you.